aristanetworks/ansible-avd
# Test: Update molecule scenario evpn_underlay_isis_overlay_ibgp input to 4.0.0

## Change Summary

Update molecule scenario evpn_underlay_isis_overlay_ibgp input to 4.0.0

## Related Issue(s)

Fixes #2920 

## Component(s) name

Molecule evpn_underlay_isis_overlay_ibgp

## How to test
run molecule converge -s evpn_underlay_isis_overlay_ibgp -- -v and ensure there is no warning regarding data conversion

## Checklist

### User Checklist


- N/A

### Repository Checklist
